### ISSUE TITLE
Build fails on OpenBSD 7.1

### ISSUE THREAD

**Issue by tarbomb** [Cogmasters/concord#104]
## Describe the bug
There is a bit of a problem when building on OpenBSD, which seems to be from the codecs generation. 
This happens on a fresh OpenBSD installation, with all dependencies for Concord installed.

## Expected behavior
The expected behavior is that the build should complete correctly-- it does not.

## Screenshots
[Here](https://pastebin.com/HucwEhJj) is a textual error log, as well as some screenshots.
![image](https://user-images.githubusercontent.com/85056428/191611110-57748af5-27f3-41ae-80dd-b68a5e7d8b39.png)

## To Reproduce
Install a new OpenBSD virtual machine, pkg_add git, clone Concord, and perform `make`

## Version
072e414c4a5cb4c686ebbeff4eafa21acd6975a7

**Comment by tarbomb** [comment]
So first things first, looking at line 5794 of `discord_codecs.h`, it seems like the token pasting is _not_ being performed.

```c
void strings##_cleanup(struct strings *self);
```

I decided to try running the Makefile in `gencodecs/` manually, got the same error. Then, I ran it as such:
`make CPP="cc -E"`, and it built fine. We may want to change the default CPP variable from `cpp` to `cc -E`, perhaps?
